consoles/vnc_base.pm

```diff
@@ -59,7 +60,7 @@ sub connect_remote {
 
     $self->{mouse} = {x => -1, y => -1};
 
-    CORE::say __FILE__. ":" . __LINE__ . ":" . bmwqemu::pp($args);
+    bmwqemu::diag "Establishing VNC connection to $args->{hostname}:$args->{port}";
```

Comment:
I think we should decide to either import the function and call it via `diag()` or not import it and call it fully qualified.

Reply:
right, done